<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowStatistics</class>
 <widget class="QWidget" name="WindowStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>301</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>631</width>
    <height>301</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>301</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>:/Image/Statistics.png</selectedon>
   </iconset>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>301</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>761</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>521</width>
      <height>221</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>221</height>
     </size>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tabFirst">
     <attribute name="title">
      <string>First Binary Tree</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>741</width>
        <height>151</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectColumns</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>180</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>25</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <row>
       <property name="text">
        <string>N - 100</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>N - 200</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>N - 300</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>N - 400</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>N - 500</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Checksum</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>The height is actual</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Theoretical assessment for medium height</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSecond">
     <attribute name="title">
      <string>Second Binary Tree</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>741</width>
        <height>151</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectColumns</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>180</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>25</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>25</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <row>
       <property name="text">
        <string>N - 100</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>N - 200</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>N - 300</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>N - 400</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>N - 500</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Checksum</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>The height is actual</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Theoretical assessment for medium height</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
